### PR TITLE
Jaleel williams

### PR DESCRIPTION
**Pull request by treatwill**
@ayunas here is My week 2 Assignment

### CODE REVIEW

**Comment by treatwill** [comment]
@tek-abrjablonski here are my week 2 assignments

**Comment by treatwill** [comment]
@mchoat2021 here are my week 2 assignments. please provide me with feedback

File: dom/inject/inject.js [hunk 1]
```diff
@@ -1,4 +1,37 @@
...
+injectDiv.appendChild(p3);
+
+
+h1.innerHTML = '<i>HTML Practice Exercise TEKcamp.</i>';
+h2.innerHTML = '<a href="">TEKsystems "TEKcamp"</a>';
+p.innerHTML = 'I love <i>HTML</i> for the following reasons:';
+ol.innerHTML = '<li>I learned it quickly.</li><li>I can make web pages using </li><li> It’s fun.</li>';
+p2.innerHTML = 'My instructor’s email address is: <a href="mailto:ayunas@teksystems.com">ayunas@teksystems.com</a>.';
+h1b.innerHTML = 'Have a great day!';
+p3.innerHTML = 'I really look forward to learning how to code!  Have a great day.  -[Team "Git\'er Done"]';
```
Comment (ghost) [inline dom/inject/inject.js:37]: Looks great. I see no issues with this inject code.

File: dom/shopping/shoppingList.js [hunk 2]
```diff
@@ -38,4 +38,36 @@
...
+
+ul.appendChild(li);
+
+    btn.onclick = function(e) {
+    ul.removeChild(li);
+  }
+
+input.focus();
+ 
+});
```
Comment (ghost) [inline dom/shopping/shoppingList.js:73]: Clean code. Nice work on the shopping list!

File: dom/timer/timer-dom.js [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+function counter() {
+
+    seconds++;
+
+    if(seconds / 60 == 1){
+        seconds = 0;
+        minutes++;
+    }
+
+    if(seconds < 10){
```
Comment (ghost) [inline dom/timer/timer-dom.js:19]: We generally do not want to stack "if" statements. It's more acceptable to use one "if" statements followed by "if else" and finally, "else". stacked "if" statements can cause bugs.

File: exercise-1.js [hunk 4]
```diff
@@ -4,31 +4,45 @@
...
 
         //your code...
-        function tekCamp() {}
-        // tekCamp();
+        function tekCamp() {
+            for (var x = 1; x <= 100; x++) {
+                if ( x%3 === 0 && x%5 === 0) {
+                    console.log(x + "TEKcamp");
+                }
+                else if (x%3 === 0) {
```
Comment (ghost) [inline exercise-1.js:12]: Good use of "else if" here instead of stacked "if" statements.

File: exercise-1.js [hunk 5]
```diff
@@ -4,31 +4,45 @@
...
 
         //your code...
-        function farenheitCelsius() {
-
-        }
+        
 
+    function fahrenheitCelsius(fahrenheit) {
+        return Math.round((fahrenheit - 32) * (5 / 9));
+     }
```
Comment (ghost) [inline exercise-1.js:35]: I don't see any example outputs here. It's important to add some test cases printed out so we can see that your function actually works.

File: exercise-1.js [hunk 6]
```diff
@@ -4,31 +4,45 @@
...
 
         //your code...
-        function celsiusFarenheit() {
-
-        }
-
-
+        function celsiusFahrenheit(celsius) {
+            return Math.round(celsius * (9 / 5) + 32);
+         }
```
Comment (ghost) [inline exercise-1.js:46]: See my comment about test cases above.

File: exercise-1.js [hunk 7]
```diff
@@ -37,8 +51,14 @@
...
         function canVote(age) {
-            return false;
+            if (age >= 18) {
+                return true;
+            }
+            else {
+             return false;
+            }
         }
+        canVote()
```
Comment (ghost) [inline exercise-1.js:62]: Again, you haven't included any example outputs here. I shouldn't have to write my own console.logs just to test if these functions work.

File: exercise-1.js [hunk 8]
```diff
@@ -51,8 +71,11 @@
 
         //your code...
         function strToArr() {
-            return [];
-        }
+            var string = "hi";
```
Comment (ghost) [inline exercise-1.js:74]: Your example string here is too short. How is your split function going to work if there is only 1 word to split? The entire point of this problem is to split a string with multiple words into an array with multiple values. Nothing is wrong with your actual function but, again, I can't verify that it actually works with longer strings unless I change your example string to be longer.

File: exercise-1.js [hunk 9]
```diff
@@ -65,17 +88,28 @@
...
+   
+           
+          function showCar(make, model, year, color) {
+            this.make = make;
+            this.model = model;
+            this.year = year;
+            this.color = color;
+          }
+         
+         const Jaleel = new showCar('Jeep', 'wrangler', 2012, 'silver');
```
Comment (ghost) [inline exercise-1.js:110]: Good job with this problem.

File: exercise-1.js [hunk 10]
```diff
@@ -88,32 +122,88 @@
...
+
+
+
         const adjectiveArray = [ 'salty', 'spicy', 'sour', 'sweet', 'rich','creamy','amazing'];
         // Using both the foodArray and the adjectiveArray, write "for" loop that console.log()'s a sentence for each corresponding value in the arrays. Add the word "is" or "are" depending on if the food is singular or plural.  i.e. "Potatoes are salty", "Lemon is sour".
 
 
+
+
+       /* let i = 0;
```
Comment (ghost) [inline exercise-1.js:182]: Thanks for showing your work here.

File: exercise-1.js [hunk 11]
```diff
@@ -122,14 +212,28 @@
 
         //your code...
 
+        
+        var h = 0;
```
Comment (ghost) [inline exercise-1.js:216]: variable names here are not descriptive enough.

File: exercise-1.js [hunk 12]
```diff
@@ -122,14 +212,28 @@
...
+        }
 
-
+    }
         /************************************************************* */
         //Multiply the sum of 30 added to two by 20.  Divide the product by 10 raised to the power of 2
         //use javascript to compute the value of the above statement. Each individual operation needs to be a function expression. run all the functions after defining them, and print the answer to the console.
 
         //your code...
+       function math() {
```
Comment (ghost) [inline exercise-1.js:228]: Doesn't follow instructions. Each individual operation needs to be a function expression. You've put them all in one function here.

File: exercise-1.js [hunk 13]
```diff
@@ -150,26 +254,43 @@
         // undefined
         // ""
 
-
+        function tOrF(val) {
```
Comment (ghost) [inline exercise-1.js:257]: I'm confused here. You're supposed to be testing each value between lines 244 to 255 and then describing why each value is either truthy or falsy.

File: exercise-1.js [hunk 14]
```diff
@@ -150,26 +254,43 @@
...
-            console.log("tuesday's are still beterr than mondays, but LONG way to go still");
-        } else if (day === "wednesday") {
-            console.log("We are smack dab in the middle of the week");
-        } else if (day === "thursday") {
-            console.log("Thursday night... the mood is right");
-        } else if (day === "friday") {
-            console.log("TGIF.  Friday truly is the best day of the week!")
-        } else {
-            console.log("It's a weekend!")
+        switch(day) {
```
Comment (ghost) [inline exercise-1.js:276]: Good work on this problem.

File: exercise-1.js [hunk 15]
```diff
@@ -178,17 +299,17 @@
         // Refactor the following statements to use ternary expressions:
 
         const age = 10;
-        if (age > 21) console.log("adult"); else {
```
Comment (ghost) [inline exercise-1.js:181]: Maybe next time just comment out the original statements instead of deleting them completely.

File: exercise-1.js [hunk 16]
```diff
@@ -208,8 +329,27 @@
         */
 
         //your code...
+        var myLife = {
+            name: 'Jaleel',
+            age: 26,
+            gender: 'male',
+            healthy: 'yes',
+            hobbies: 'games and sports',
```
Comment (ghost) [inline exercise-1.js:337]: What games do you like to play? What about sports? I used to play Ice Hockey and Baseball. I tried basketball but I'm way too short!

File: exercise-1.js [hunk 17]
```diff
@@ -224,9 +364,9 @@
 
 
             // 1.
-            if(year > 2000 && year < 2100) {
+            var century = (year > 2000) && (year < 2100) ? "welcome to 21st century" : "you're lost dude!";
```
Comment (ghost) [inline exercise-1.js:367]: Good work here.

File: exercise-1.js [hunk 18]
```diff
@@ -386,53 +563,66 @@
...
+        console.log(notDevs);
 
         /************************** */  
         // Calculate the total age of all the devs
 
         //your code here...
-
-
+        
+        //GET HELP!!!
```
Comment (ghost) [inline exercise-1.js:583]: Did you not attempt this part?

File: exercise-1.js [hunk 19]
```diff
@@ -386,53 +563,66 @@
...
+        console.log(femaleCode);
+    
 
         /************************** */  
         // lowercase the genders of every dev
 
         //your code here...
-
+      // GET HELP FROM AMIR!!!1!
+
```
Comment (ghost) [inline exercise-1.js:598]: Did you google for this problem? The toLowerCase function may have been the solution you needed here.

**Review by ghost — APPROVED** [review]
Good work on these assignments. See my comments for exercise 1 and let me know if you want to discucss them further. Thanks!

File: exercise-1.js (hunk 16 above)
Comment (ayunas) [reply exercise-1.js:337]: yea @treatwill  looking for an array of hobbies that you actually enjoy.  

@mchoat2021 Ice hockey would have been fun during the winter storm.  I think a tekcamper from last bootrcamp did play ice hockey this week.  He Found a frozen pond and started sliding the puck.

**Pull request merged**